pkg/config/config_template.yaml

```diff
@@ -217,7 +218,40 @@ api_key:
 #
 # The timeout to execute the command in second
 # secret_backend_timeout: 5
-
+#
+{{ end -}}
+{{- if .LogsAgent }}
+# Logs agent
+#
+# Enable logs collection for all containers, disabled by default
+# logs_enabled: false
+#
+# logs_config:
+#
+# Enable container log collection for all the containers (see ac_exclude to filter out containers)
+#   container_collect_all: false
+#
+# Define the endpoint and port to hit when using a proxy for logs. The logs are forwarded in TCP
+# therefore the proxy must be able to handle TCP connections.
+#   logs_dd_url: <endpoint>:<port>
+#
+# Disable the SSL encryption (default to false). This parameter should only be used when logs are
+# forwarded locally to a proxy. It is highly recommended to then handle the SSL encryption 
+# on the proxy side. 
+#   logs_no_ssl: false
+#
+# Global processing rules that are applied to all the logs. The available rules are
+# "exclude_at_match", "include_at_match" and "mask_sequences"
+#   processing_rules:
+#     - rule1_arg1
+#       rule1_arg2
+#     - rule2_arg1
+#       rule2_arg2
+#
+# By default, logs are forwarded on the port 10516 (for the US endpoint), use this parameter
+# to force the agent to forward logs in TCP through the port 443 (default is false)
+#   use_port_443: false
```

Comment:
I think this option should be grouped with the other "proxy-related" options above for clarity

Reply:
Definitely not linked with the proxy and I want to avoid this confusion.

Usually people believe that using this port 443 means that it becomes HTTP and can go through an HTTP proxy for it. This parameter is used to send data through port 443 but not to a proxy.
Directly to Datadog.

Which is why I have separated this option from the proxy section.

Reply:
oh, thanks for the explanation, disregard then 👍